src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -92,7 +92,7 @@ def test_reset_device(session):
 def test_reset_with_default(session):
     channel = session.channels['0']
     assert channel.aperture_time_units == nidcpower.ApertureTimeUnits.SECONDS
```

Comment:
The code could be improved by putting the constant on the left side, thus making this kind of copy/paste error less frequent.